.clabot

```diff
@@ -1,5 +1,5 @@
 {
   "contributors": "https://api.github.com/repos/trinodb/cla/contents/contributors",
-  "message": "Thank you for your pull request and welcome to the Trino community. We require contributors to sign our [Contributor License Agreement](https://github.com/trinodb/cla/raw/master/Trino%20Foundation%20Individual%20CLA.pdf), and we don't seem to have you on file. Continue to work with us on the review and improvements in this PR, and submit the signed CLA to cla@trino.io. Processing may take a few days. The CLA needs to be on file before we merge your changes. For more information, see https://github.com/trinodb/cla",
+  "message": "Thank you for your pull request and welcome to the Trino community. We require contributors to sign our [Contributor License Agreement](https://github.com/trinodb/cla/raw/master/Trino%20Foundation%20Individual%20CLA.pdf), and we don't seem to have you on file. Continue to work with us on the review and improvements in this PR, and submit a scan or a photo of the signed CLA to cla@trino.io. Processing may take a few days. The CLA needs to be on file before we merge your changes. For more information, see https://github.com/trinodb/cla",
```

Comment:
It can also be signed digitally with a PDF editor (Adobe, Preview, etc)

Reply:
Maybe it would be better to add a separate sentence. 

```
submit the signed CLA to cla@trino.io. Photos, scans, or digitally signed PDF files are all suitable.